aqua/qt5/Portfile

```diff
@@ -551,12 +551,12 @@ array set modules {
     }
     qttools {
         {
-            35f435253529301da39d017931da4f3c1778fd04
-            5bb0cf7832b88eb6bc9d4289f98307eb14b16a453ad6cf42cca13c4fe1a053c5
-            8815020
+            32d7ac88f4769c4b6a4b57082af29a8f8c6c1915 \
+            c189d0ce1ff7c739db9a3ace52ac3e24cb8fd6dbf234e49f075249b38f43c1cc
+            8894728
         }
         ""
-        "port:clang-9.0"
```

Comment:
This was here because of some clang library that qttools needed, apparently. Was it removed just because it was not building, or was it removed because it was no longer needed? If it is a building issue,  clang-11 at least builds on arm64. I don't recall just now if clang-10 did as well.

Reply:
yes, a clang port is required if the documentation is to be built. clang-11 would be the obvious one, as it builds on all systems that can run qt-5.15.2 anyway.
```
with clang-11:

Running configuration tests...
Checking for libclang... yes
Qt Tools:
  QDoc ................................... yes

--- 

without clang-11:

Running configuration tests...
Checking for libclang... no
Qt Tools:
  QDoc ................................... no

WARNING: QDoc will not be compiled, probably because libclang could not be located. This means that you cannot build the Qt documentation.